### ISSUE TITLE
add Carousel component

### ISSUE THREAD

**Issue by gioboa** [qwikifiers/qwik-ui#124]

**Comment by luisbeqja** [comment]
I can work on this!

**Comment by gioboa** [comment]
I saw your message in the Discord Server and in my opinion we can use a dependency to solve particular needs.
I would like to know what other maintainers think. @shairez @gilf @nnelgxorz

**Comment by shairez** [comment]
@luisbeqja @gioboa 
saw this  -
https://codepen.io/jh3y/pen/WwVKLN

purely implemented with CSS
is it relevant for us? 

Plus - a carousel has a lot in common with other components ([as presented in this lecture](https://www.youtube.com/watch?v=mtHf7crZZIQ))
so another thing to consider that if we add a dependency we bind ourselves to that api

but then again, I never used swiper.

What do you think are the pros / cons of using it?

**Comment by tleperou** [comment]
Huge fan of pure css solution here 🙌 

The [scroll APIs](https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior) bring great opportunities in developing html-first _carousel_ alike components.

[Swiffyslider](https://swiffyslider.com/) is an interesting example.

**Comment by gioboa** [comment]
Thanks for the example @tleperou 👏  So let's try the no dependency approach.
@luisbeqja or @tleperou would you like work on it?

**Comment by luisbeqja** [comment]
@gioboa these days I'm a little bit full of work, if @tleperou would like to work on this for me it is ok

**Comment by tleperou** [comment]
My top priority (for my needs 🤷 ) is user's input.

Meanwhile, I worked super recently on this matter. I'm OK to get into it 👍 

Do you have a task manager to track this out ?

**Comment by gioboa** [comment]
No, just issue assignment. If you need an help you can open a squad on discord server about this task

**Comment by thejackshelton** [comment]
Closing, as this was added in draft status.
